### PR TITLE
Mean issue-claim age chart

### PR DESCRIPTION
**Pull request by whaught**
Fixes https://github.com/google/exposure-notifications-verification-server/issues/1231

## Proposed Changes

* Shows a line chart of average issue-claim age

**Release Note**

```release-note
Average issue-claim chart
```

### CODE REVIEW

File: pkg/database/migrations.go [hunk 1]
```diff
@@ -2087,8 +2087,7 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 			ID: "00093-AddClaimMeanAgeToRealmStats",
 			Migrate: func(tx *gorm.DB) error {
 				return multiExec(tx,
-					`ALTER TABLE realm_stats ADD COLUMN IF NOT EXISTS code_claim_mean_age BIGINT NOT NULL DEFAULT 0`,
-					`ALTER TABLE realm_stats DROP COLUMN IF EXISTS codes_claimed_age_distribution`) // result of previous mistake
+					`ALTER TABLE realm_stats ADD COLUMN IF NOT EXISTS code_claim_mean_age BIGINT NOT NULL DEFAULT 0`)
```
Comment (sethvargo) [inline pkg/database/migrations.go:2090]: Is this intended?

Comment (whaught) [reply pkg/database/migrations.go:2090]: Yea I think it's confusing to leave in. It only got deployed to dev

**Pull request merged**